src/Utils/getSignalLabel.ts

```diff
@@ -2,9 +2,24 @@ export interface CollectorSignals {
   primaryLabel?: string | null | undefined
 }
 
-export const getSignalLabel = (collectorSignals: CollectorSignals) => {
+interface SignalLabelProps {
+  collectorSignals: CollectorSignals
+  hideSignals?: boolean
+}
+
+export const getSignalLabel = ({
+  collectorSignals,
+  hideSignals,
+}: SignalLabelProps) => {
   const { primaryLabel } = collectorSignals
 
+  if (
+    hideSignals &&
+    (primaryLabel === "CURATORS_PICK" || primaryLabel === "INCREASED_INTEREST")
+  ) {
+    return ""
```

Comment:
If this function is already returning an empty string as a false-like value, can the places we call is just pass in their possibly null or undefined `collectorSignals` and get the same value back, rather than having to do [this](https://github.com/artsy/force/pull/14661/files?diff=unified&w=1#diff-6bf5e8fb17b2e6befb4c59e435defb597806249577220e33a8dca47b8589dcdaL64-R68)?